apps/sms/test/marionette/lib/messages.js

```diff
@@ -431,6 +440,21 @@
 
             window.wrappedJSObject.TestStorages.setStorageReady('messagesDB');
           }, [threads || [], uniqueMessageIdCounter || 0]);
+        },
+
+        /**
+         * Sets pre-populated contacts storage.
+         * @param {Array.<MozContact>?} contacts List of the contacts to
+         * pre-populate storage with.
+         */
+        setContactsStorage: function(contacts) {
```

Comment:
nope, for the 2 methods about storage, because I feel like they're not manipulating the Messages app's interface, so maybe they should not be in the same object than other interface-manipulating functions...

but nothing mandatory here, it's only a feeling.